src/components/toast/toast.js

```diff
@@ -30,14 +30,14 @@ export const props = {
     type: String,
     default: null
   },
-  variant: {
-    type: String,
-    default: null
-  },
   toaster: {
     type: String,
     default: () => getComponentConfig(NAME, 'toaster') || 'b-toaster-top-right'
```

Comment:
Please remove that too.

Reply:
Although a toaster name is required... and users could set it to null or an empty string by mistake.

Reply:
But it adds additional bundle size, which de did with the config stuff already enough.

Reply:
I could force it to a string

Reply:
This will prevent the prop type warning for invalid value in DEV-mode.
Not sure about that change.

Reply:
I'm not sure the prop type check runs on the `default` value or not.

Reply:
But it will prevent PortalTarget from crashing ;-)

Reply:
Unless we add a computed prop that stringifies the name, and we pass that computed prop toPortal and PortalTarget